ast/lexer.mll

```diff
@@ -98,6 +98,27 @@ let keyword_table =
     "asr", INFIXOP4("asr")
 ]
 
+module Buffer = struct (* Imported for compatibility with 4.04.x *)
+  include Buffer
```

Comment:
I think the following pattern is slightly better as it uses the upstream functions for newer version of OCaml:

```
[@@@warning "-32"]
let add_utf_8_uchar = ...
include Bufffer
```

Reply:
Sure